RecommendedAutomations.md

```diff
@@ -0,0 +1,45 @@
+# Recommended Automations
```

Comment:
maybe we should do something more natural like "integrations"
and add "integration with bridgecrew.cloud", "integration with checkov" integration with "parliament" etc...